3.11/release-notes-api-changes311.md

```diff
@@ -63,6 +63,15 @@ The cursor API can now return an additional statistics value in its `stats` sub-
   In a cluster, the intermediate commits are tracked per DB server that participates in the query
   and are summed up in the end.
 
+The cursor API can now receive a retry request to retrieve the response for the latest batch. 
+The response object for `_api/_cursor/<cursorId>` now contains a sub-attribute `nextBatchId` which is 
+the id of the next batch that will be fetched when the cursor advances, after executing the current 
+request. Then, on the next run of `_api/_cursor/<cursorId>`, which should output the response object 
+of the batch with `nextBatchId`, if it's unsuccessful because of some connection issue, the user can 
+retry to get the response object for that batch with a POST request to `_api/<cursorId>/<nextBatchId>`.
```

Comment:
```suggestion
The response object for `/_api/_cursor/<cursorId>` now contains a sub-attribute `nextBatchId` which is 
the id of the next batch that will be fetched when the cursor advances, after executing the current 
request. Then, on the next run of `/_api/_cursor/<cursorId>`, which should output the response object 
of the batch with `nextBatchId`, if it's unsuccessful because of some connection issue, the user can 
retry to get the response object for that batch with a POST request to `/_api/<cursorId>/<nextBatchId>`.
```